src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/GenerateWasmBootJson.cs

```diff
@@ -211,13 +223,48 @@ public void WriteBootJson(Stream output, string entryAssemblyName)
                     resourceList = resourceData.runtime;
                 }
                 else if (string.Equals("JSModule", assetTraitName, StringComparison.OrdinalIgnoreCase) &&
-                            string.Equals(assetTraitValue, "JSLibraryModule", StringComparison.OrdinalIgnoreCase))
+                    string.Equals(assetTraitValue, "JSLibraryModule", StringComparison.OrdinalIgnoreCase))
                 {
                     Log.LogMessage(MessageImportance.Low, "Candidate '{0}' is defined as a library initializer resource.", resource.ItemSpec);
-                    resourceData.libraryInitializers ??= new();
-                    resourceList = resourceData.libraryInitializers;
+
+                    if (TargetingNET80OrLater)
```

Comment:
Is this block of code adding some new behavior? Can we name the property to represent that new feature/behavior?

Reply:
If not, then take the version as input, and decide the behavior within the task.

Reply:
What is the benefit of such indirection? I see it easier to read the code when using single property to distinguish version specific behavior.

Reply:
You can have local named `TargetingNET80OrLater` set based on the version received as input.